frotz.py

```diff
@@ -235,61 +283,63 @@ def _configure_vte(self):
         else:
             bg_color = '#FFFFFF'
             conf.set('terminal', 'bg_color', bg_color)
-        self.set_colors(gtk.gdk.color_parse (fg_color),
-                            gtk.gdk.color_parse (bg_color),
-                            [])
-                            
+
+        self.set_colors(self.color_parse(bg_color),
+                        self.color_parse(fg_color), [])
+
         if conf.has_option('terminal', 'cursor_blink'):
             blink = conf.getboolean('terminal', 'cursor_blink')
         else:
-            blink = False
+            blink = 'False'
             conf.set('terminal', 'cursor_blink', blink)
-        
-        self.set_cursor_blinks(blink)
+        blink = Vte.CursorBlinkMode(0) if blink == 'False' else Vte.CursorBlinkMode(1)
+        self.set_cursor_blink_mode(blink)
 
         if conf.has_option('terminal', 'bell'):
             bell = conf.getboolean('terminal', 'bell')
         else:
-            bell = False
+            bell = 'False'
```

Comment:
Should be a boolean.

Reply:
Because` configparser accepts <class 'str'> as *args.`

Reply:
Not a good enough reason.  What matters here is what Vte requires.

Reply:
https://github.com/sugarlabs/Frotz/pull/7/commits/813abae39151c64aebca002642f97413fc204c07 Thanks Please review.